lib/IDE/Refactoring.cpp

```diff
@@ -2601,6 +2601,77 @@ bool RefactoringActionLocalizeString::performChange() {
   EditConsumer.insertAfter(SM, Target->getEndLoc(), ", comment: \"\")");
   return false;
 }
+    
+bool RefactoringActionDocCommentBoilerplate::isApplicable(ResolvedCursorInfo Tok, DiagnosticEngine &Diag) {
+#warning Implement here
+    if (Tok.Kind != CursorInfoKind::ValueRef)
+        return false;
+    
+    return Tok.ValueD->getKind() == DeclKind::Func;
+}
+    
+bool RefactoringActionDocCommentBoilerplate::performChange() {
+#warning Implement here
+    if(CursorInfo.ValueD->getKind() != DeclKind::Func) return true;
+    
+    if(!isa<FuncDecl>(CursorInfo.ValueD)) return true;
+    
+    FuncDecl *funcDecl = (FuncDecl *)CursorInfo.ValueD;
+    
+    swift::SourceLoc startLocation = CursorInfo.ValueD->getStartLoc();
+    EditConsumer.accept(SM, startLocation, "///<#Function Summary#>\n");
```

Comment:
Last little nitpick, though this is looking great, can you add a space after `///`?